tests/MercadoPago/Client/PreApprovalPlan/PreApprovalPlanClientUnitTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace MercadoPago\Client\PreApprovalPlan;
+
+use MercadoPago\MercadoPagoConfig;
+use MercadoPago\Net\MPDefaultHttpClient;
+use MercadoPago\Client\Base\BaseClient;
+
+/**
+ * PreApprovalPlan Client unit tests.
+ */
+final class PreApprovalPlanClientUnitTest extends BaseClient
+{
+    public function testCreateSuccess(): void
+    {
+        $filepath = '../../../Resources/Mocks/Response/PreApprovalPlan/preapprovalplan_base.json';
```

Comment:
Alguns lugares essa variável esta escrita dessa forma: $file_path. Como estamos usando o padrão snake case ajustar os testes que estão dessa .
```suggestion
        $file_path = '../../../Resources/Mocks/Response/PreApprovalPlan/preapprovalplan_base.json';
```

Reply:
Bom ponto. Pelo jeito escrevemos de maneira distinta em vários arquivos, acho que vale a pena ajustar isso em outro PR para não poluir esse.